assignment2.js

```diff
@@ -0,0 +1,26 @@
+/**
+*
+* Assume we are fetching the data from a rest endpoint in the get data function.
+* we can simulate the same by replacing the setTimeout with fetch api and executing the same in a browser.
+*
+*/
+async function getData(user_Id) {
+    const promise = new Promise(resolve => {
+        setTimeout(() => {
+            console.log("Fetched the data!");
+            return resolve(user_Id+"@gmail.com");
+        }, 4000);
+    });
+    return promise;
+}
+
+
+const emailGET = async () => {
```

Comment:
getEmail is better understandable than emailGET

Reply:
Yes I will modifiy it.